conf/parse.go

```diff
@@ -84,18 +87,18 @@ func ParseFile(fp string) (map[string]interface{}, error) {
 	return p.mapping, nil
 }
 
-func ParseFileWithChecks(fp string) (map[string]interface{}, error) {
+func ParseFileWithChecks(fp string) (map[string]interface{}, []string, error) {
```

Comment:
supposed ok to change this although public since there has not been a release (also there might be a better name of the function, the main difference is that this version of `ParseFile` collects more metadata from the parsing).

Reply:
Agree.